godsenttools_gpu_saver/resource/localization/fr/godsenttools_gpu_saver.properties

```diff
@@ -0,0 +1,9 @@
+godsenttools.gpusaver.name=Économiseur vidéo
+godsenttools.gpusaver.description=L'économiseur vidéo désactive entièrement le rendu lorsque la fenêtre du jeu n'est pas focalisée ou réduite. Cela permet de réduire la charge du processeur graphique et peut-être de vous épargner un crash de la mémoire virtuelle.
+godsenttools.gpusaver.osxwarning=Attention ! MacOS détecté, cet addon ne fonctionne pas avec ce système d'exploitation. L'addon sera automatiquement désactivé. Vous pouvez toujours utiliser cet addon avec le bouton « Activer manuellement ».
```

Comment:
Do you really put spaces before `!` and `:` in your language?
Same question about spaces between « »

Reply:
Yes, this is a specification of the French language: https://en.wikipedia.org/wiki/History_of_sentence_spacing#French_and_English_spacing